tests/smoketests/test_Project.py

```diff
@@ -528,6 +527,7 @@ def test_subtable(self, example_pep_cfg_path):
         Verify that the sample merging takes place
         """
         p = Project(cfg=example_pep_cfg_path)
+        a = p.samples
```

Comment:
Maybe something more verbose than `a` as variable name? :D

Reply:
we don't need it. I forgot to delete this line